python/svix/api/api.py

```diff
@@ -2,32 +2,32 @@
 from contextlib import contextmanager
```

Comment:
This is a very confusing name for this file. :)

I think you don't need to change it though. Just keep as it was.